src/utils/tooltipify.tsx

```diff
@@ -39,30 +39,23 @@ export function tooltipifyLinks(rootNodes: ArrayLike<Element>, ignoredNodes: Ele
     let node = rootNodes[0];
 
     while (node) {
-        let tooltipified = false;
-
-        if (ignoredNodes.indexOf(node) >= 0) {
+        if (ignoredNodes.includes(node) || containers.includes(node)) {
             node = node.nextSibling as Element;
             continue;
         }
 
         if (node.tagName === "A" && node.getAttribute("href")
             && node.getAttribute("href") !== node.textContent.trim()
         ) {
-            const container = document.createElement("span");
             const href = node.getAttribute("href");
 
             const tooltip = <LinkWithTooltip tooltip={new URL(href, window.location.href).toString()}>
-                { node.innerHTML }
+                <span dangerouslySetInnerHTML={{ __html: node.innerHTML }} />
```

Comment:
can we get a comment which cites where the node's safe HTML comes from? (so we have a chance of not regressing it when we do a refactor)